ballerina/tests/accounts_tests.bal

```diff
@@ -0,0 +1,99 @@
+// Copyright (c) 2024, WSO2 LLC. (http://www.wso2.com).
+//
+// WSO2 LLC. licenses this file to you under the Apache License,
+// Version 2.0 (the "License"); you may not use this file except
+// in compliance with the License.
+// You may obtain a copy of the License at
+//
+// http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing,
+// software distributed under the License is distributed on an
+// "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+// KIND, either express or implied.  See the License for the
+// specific language governing permissions and limitations
+// under the License.
+
+import ballerina/test;
+
+isolated string commonAccountId = "";
+
+function setCommonAccountId(string accountId) {
+    lock {
+        commonAccountId = accountId;
+    }
+}
+
+function getCommonAccountId() returns string {
+    lock {
+        return commonAccountId;
+    }
+}
+
+@test:Config {
+    groups: ["live_tests", "mock_tests"]
```

Comment:
Do we need these groups in test cases?

Reply:
Yes, AFAIR this is the practice we agreed right ? IIRC the plan was to run the `live_tests` as a group in the workflow.

Reply:
As I remember, we have a single test suite. And when the env `IS_LIVE_SERVER` is set to true, it runs against an actual endpoint otherwise it just runs on the mock server. Therefore we don't need two separate test groups. Correct me if I'm wrong.

Reply:
@Nuvindu The initial idea was to separate the two test suites using test groups, allowing us to have different paths for both scenarios. However, in this case, we need to initialize the client based on the group type since the connection configurations differ. But unfortunately, there was no way to detect the test group being passed to the CLI command at runtime.

This limitation led us to introduce the `IS_LIVE_SERVER` environment variable approach, which is a less prominent solution compared to the test groups, IMO

I’m +1 for keeping the test groups ftm, in case we'll the support to detect the test group being passed to the CLI command at runtime in the future. 

@Dilhasha appreciate your input on the possibility of supporting the above requirement.

Reply:
@NipunaRanasinghe there's no immediate plan to support this requirement. It'd be great if you can create an issue mentioning the requirement. We'll check the possibility of prioritizing.